docs/production-deployment/cloud/billing-and-cost.mdx

```diff
@@ -30,49 +30,92 @@ tags:
   - invoice history
 ---
 
-The Account Owner and Finance Admin hold the ability to access and view detailed billing and cost information at any time, ensuring full transparency and control over financial aspects. If you need to assess your spending patterns, check invoice histories, or handle payment details, Temporal Cloud's Cost and Billing capabilities are able to provide the information you need.
+Temporal strives to provide full transparency over billing and costs.
+Account Owners and Finance Admins can view their [detailed billing information](https://cloud.temporal.io/billing) at any time.
+Use this information to assess your spending patterns, inspect your credit ledger, check your invoice histories, or update payment details as needed.
 
-For more information on Temporal Cloud pricing for actions, storage, and services/support, view [Pricing](/cloud/pricing).
+For more information on Temporal Cloud pricing for Actions, storage, and services/support, please visit our [Pricing](/cloud/pricing) page.
 
-On the **Billing** page, you can find the following details:
+Your [Billing](https://cloud.temporal.io/billing) page includes all or some of the following information, based on your plan:
 
-- [Current Balance](#current-balance): Your balance accrued thus far in the billing cycle
+- [Current balance](#current-balance): Your balance to date for this billing cycle
 - [Recent bill](#recent-bill): The amount of your most recent bill
-- [Invoice history](#invoice): A record of all past invoices
-- Credit information: If applicable, details about any credit grants, purchases, usage, and remaining credits.
+- [Invoices](#invoice): Access to all past invoices
+- [Credits](#credit-table): A record of all credit related transactions.
+  This includes details on credit grants, purchases, usage, and remaining credits, if applicable.
 
-If you are not on a standard plan, the Billing page may not include all of the information stated above. 
+Your [Usage](https://cloud.temporal.io/usage) provides an overview of your usage across your Namespaces.
+In addition, Account Owners can review [cost breakdowns](#cost-by-namespace) by Namespace.
 
-## How to navigate to your invoices
+## Current balance {#current-balance}
+
+Your current balance card shows the balance for your current billing cycle and the date it was last updated.
+This balance adjusts with use and appears on the first line of your Invoices table.
+
+:::note Billing Cycles
+
+Billing cycles normally begin on the first of the month (UTC).
+The minimum support fee for your first month is prorated based on your sign-up date.
+
+:::
+
+## Recent bill {#recent-bill}
+
+The "Recent Bill" card displays the previous bill amount.
+
+![Recent bill card showing a balance of $0.00](/img/billing/billing-card.png)
+
+* If you are paying invoices through the Stripe portal, you'll also see a **Pay Now** button, which takes you to the Stripe portal to complete your payment.
```

Comment:
? Is there a typo here?

If you are paying invoices through the Stripe portal, you're already on the Stripe portal. But you mention that if you click the Pay Now button, it takes you to the Stripe portal...which you're already on?

Reply:
Maybe rewording would help? If you pay your invoices through Stripe, you'll also see...

Reply:
Fixed